### ISSUE TITLE
Show number of tiles in enemy kingdoms.

### ISSUE THREAD

**Issue by jhcole** [Sesu8642/FeudalTactics#57]
It would be helpful if you could see a count of the number of productive tiles in an enemy kingdom just by selecting that kingdom.  The information is already available to the player, but they have to count the tiles manually, which is not much fun.  Knowing whether the enemy has enough tiles to support their current army is strategically useful.  I suggest this information be shown after selecting an enemy kingdom, where it currently shows your selected kingdom's savings and income. E.g. instead of "Savings 10 (+2)" something like "Enemy Income 19".

**Comment by Sesu8642** [comment]
Good idea. I think being able to select enemy kingdoms just like your own could confuse new players. Maybe it could be shown while holding down the mouse button / finger on the enemy kingdom. Technically the enemy savings can be calculated as well and (thinking about multiplayer) cheaters could easily make a patch to display it. I think it makes sense to just display the savings as well.

**Comment by Asmageddon** [comment]
Another very good way to handle it that I can think of would be to display the number of tiles as a label under/in the corner of the city representing that kingdom.
